roles/matrix-bridge-mautrix-signal/templates/systemd/matrix-mautrix-signal-daemon.service.j2

```diff
@@ -34,7 +34,9 @@ ExecStartPre=-{{ matrix_host_command_docker }} run --rm --name matrix-mautrix-si
 # We can't use `--read-only` for this bridge.
 ExecStart={{ matrix_host_command_docker }} run --rm --name matrix-mautrix-signal-daemon \
 			--log-driver=none \
+            {% if matrix_mautrix_signal_deamon_trust_new_security_nuber %}
 			--env SIGNALD_TRUST_NEW_KEYS=true \
+            {% endif %}
```

Comment:
If Signald exposes other things to be configured via environment variables, it may be better (and cleaner) to do this using an environment variables file -- one file, where we template all of them, which is later referenced here using `--env-file ...`.

See how the Jitsi role does it for its various components.

Additional benefits of using `--env-file` instead of `--env` (besides the cleaner `.service` file) is:

- changes to environment files do not need to reload systemd (`systemctl daemon-reload`) for the changes to take effect. Simplify restarting the service will make it pick up the new environment variable values
- we can make it extensible in a clean way. See this: https://github.com/spantaleev/matrix-docker-ansible-deploy/blob/9abcb10f9fc2341bcc02da72710f53b7db255c52/roles/matrix-jitsi/defaults/main.yml#L162-L172